docs/modules/ROOT/pages/how-to/use-change-data-capture.adoc

```diff
@@ -0,0 +1,72 @@
+[[change-data-capture]]
+:description: This page describes how use Change Data Capture API.
+= Use Change Data Capture
+
+link:https://neo4j.com/docs/cdc/current/[Change Data Capture] (CDC) queries can be created with the Cypher Builder by using the Procedures in the `cdc` namespace:
+
+* `Cypher.cdc.current()`
+* `Cypher.cdc.earliest()`
+* `Cypher.cdc.query(from, selectors)`
+
+
+== Examples
+
+=== Acquiring the current change identifier
+
+To acquire the current change identifier, follow these instructions:
```

Comment:
```suggestion
In order to query changes, you need to acquire a change identifier. This can be done with `cdc.current()` or `cdc.earliest()`. To acquire the latest change identifier, follow these instructions:
```